src/Package/Impl/Plots/PlotContentProvider.cs

```diff
@@ -31,7 +31,9 @@ internal sealed class PlotContentProvider : IPlotContentProvider {
         private async void RSession_Connected(object sender, EventArgs e) {
             // Let the host know the size of plot window
             if (_lastPixelWidth >= 0 && _lastPixelHeight >= 0) {
-                await ApplyNewSize();
+                try {
+                    await ApplyNewSize();
+                } catch (OperationCanceledException) { } catch (MessageTransportException) { } catch (RException) { }
```

Comment:
Do you really want to catch `RException` here? It would indicate an actual eval failure, which is usually a bug and not expected. Or is there some known case where eval can fail here and it is expected?